packages/commonwealth/client/scripts/views/pages/NotificationSettings/index.tsx

```diff
@@ -24,6 +25,19 @@ import './index.scss';
 
 type NotificationSection = 'community-alerts' | 'subscriptions';
 
+/**
+ * Get the unique communities but also sort them.
+ */
+function getUniqueCommunities() {
```

Comment:
lets add this function to `packages/commonwealth/client/scripts/helpers/{some-helper-file}.ts`